service/src/test/resources/config/test.yml

```diff
@@ -22,3 +22,6 @@ swagger:
 logging:
     loggers:
         org.hibernate.SQL: ALL
+
+httpClient:
+    timeout: 1s
```

Comment:
Looking at the execution time of the test with these changes which passed vs older commits without any of these changes, there doesn't appear to be a significant performance drop (+/- 2 seconds over the course of a ~2 minute build). The default was 500ms and appears to have been that for years. We must have been on the cusp of hitting that timeout already.

I think the Travis CI executors are just slow. It takes them about twice as long to execute `mvn test` as does my laptop.